src/main/kotlin/org/rust/ide/console/RsConsoleCodeFragmentContext.kt

```diff
@@ -57,15 +102,17 @@ class RsConsoleCodeFragmentContext {
 }
 
 class RsConsoleOneCommandContext(codeFragment: RsReplCodeFragment) {
-    val topLevelElements: MutableMap<String, String> = mutableMapOf()
-    val statementsText: String
+    val command: String
+    val itemsNames: List<String>
```

Comment:
I am not sure that we need `Set` here:
* we don't need fast lookups (because we don't do lookups)
* we don't need uniqueness (actually if command is executed successfully then all names should already be unique)